src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -193,6 +207,46 @@ public void InitializeNonBooleanTypeOptions(DacDeployOptions options)
             }
         }
 
+        /// <summary>
+        /// Preparing all object types which are considered as boolean options
+        /// </summary>
+        public void InitializeObjectTypesDictionary()
+        {
+            Type objectTypeEnum = typeof(ObjectType);
+            string excludePrefixWord = "Exclude ";
+            foreach (string name in Enum.GetNames(objectTypeEnum))
+            {
+                MemberInfo[] member = objectTypeEnum.GetMember(name);
+                MemberInfo info = (member != null) ? member.FirstOrDefault() : null;
+                string displayName = "";
+                if (info != null)
+                {
+                    string dacfxDisplayName = info.GetCustomAttribute<DisplayAttribute>().GetName();
+
+                    //Excluding the "Exclude " word from the option display name using Substring() as all display names are prefixed with Exclude word in DacFx
```

Comment:
does the "Exclude" part of display names get localized in DacFx?

Reply:
Hmm, yeah Exclude word is localized in DacFx, If doing like this have any effect with localization?

Reply:
Since all DacFx string are like "Exclude Aggregates, Exclude Tables ,Exclude Views ...." SSDT displaying as is. But, here we are displaying without the exclude word. Either we also display it as 'Exclude' (But when we are converting them from 'exclude...' name to Object type for DacFx, we need to get the corresponding enum number from "Exclude Table"=>ObjectType[]{"Key:Table, value:number"}) or remove the exclude word from dacfx(It changes the UI options names in SSDT as well).


What do you suggest here? any other approach than this?

Reply:
As discussed Updated the DacFx display names and removed the exclude word from the object type names. 

PR for DacFx changes: https://msdata.visualstudio.com/SQLToolsAndLibraries/_git/DacFx/pullrequest/848753